src/openrct2/ReplayManager.cpp

```diff
@@ -415,23 +429,25 @@ namespace OpenRCT2
             }
         }
 
-        virtual bool StartPlayback(const std::string& file) override
+        void StartPlayback(const std::string& file) override
         {
             if (_mode != ReplayMode::NONE && _mode != ReplayMode::NORMALISATION)
-                return false;
+                throw std::invalid_argument(std::string("Unexpected mode ") + modeToName[EnumValue(_mode)]);
 
             auto replayData = std::make_unique<ReplayRecordData>();
 
-            if (!ReadReplayData(file, *replayData))
+            try
             {
-                LOG_ERROR("Unable to read replay data.");
-                return false;
+                ReadReplayData(file, *replayData);
+            }
+            catch (const std::invalid_argument&)
+            {
+                throw;
             }
 
             if (!LoadReplayDataMap(*replayData))
             {
-                LOG_ERROR("Unable to load map.");
-                return false;
+                throw std::invalid_argument("Unable to load map.");
```

Comment:
std::runtime_exception might be the better choice here, nothing suggests anything about invalid arguments here.